src/app/view/view-routing.module.ts

```diff
@@ -38,6 +38,10 @@ const viewRoutes: Routes = [
     canActivate: [AuthGuard, WorkspaceGuard, CollectionsGuard, LinkTypesGuard, ViewsLoadedGuard, ViewExistGuard],
     component: ViewComponent,
     children: [
+      {
+        path: Perspective.Detail,
+        loadChildren: './perspectives/detail/detail-perspective.module#DetailPerspectiveModule'
```

Comment:
Will the components from this perspective module never be used in any other perspective? If no, it can stay lazy-loaded. Otherwise, we will end up with strange errors in the future.

Reply:
Most of the components are in shared module. If you mean document-detail, 1) why didn't you tell me ;-), 2) it cn be moved to shared folder later, once better written.